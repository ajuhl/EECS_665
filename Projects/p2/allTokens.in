123
+
x
{     }      (      )      ;
,      .      <<     >>     ++
--     +      -      *      /
!      &&     ||     ==     !=
<      >      <=     >=     =
bool
int
void
true
false
struct
cin
cout
if
else
while
return
// this is a comment
# this is also a comment
"this"
"\n"
"\t"
"\""
"\'"
"\?"
"\\"
x
567
"that"
y
"those"
"the cat in the
he the she
"  \y"
" \u
"the cat
3431243214324213421
