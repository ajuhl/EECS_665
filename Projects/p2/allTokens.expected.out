INTLIT:123
+
ID:x
{
}
(
)
;
,
.
<<
>>
++
--
+
-
*
/
!
&&
||
==
!=
<
>
>=
>=
=
bool
int
void
true
false
struct
input
output
if
else
while
return
STRINGLIT:"this"
STRINGLIT:"\n"
STRINGLIT:"\t"
STRINGLIT:"\""
STRINGLIT:"\'"
STRINGLIT:"\?"
STRINGLIT:"\\"
ID:x
INTLIT:567
STRINGLIT:"that"
ID:y
STRINGLIT:"those"
ID:he
ID:the
ID:she
INTLIT:532915885
EOF
